.gitignore

```diff
@@ -5,3 +5,5 @@
 build/
 unicode/
 test262_*.txt
+.idea
+cmake-*
```

Comment:
what files are created with this name?

Reply:
.idea:  clion project config folder

cmake-*: clion default cmake generated artifacts folder